src/core/__tests__/findAllMatches.js

```diff
@@ -8,7 +8,7 @@ test('Should find several exact matches in a text', (t) => {
   t.deepEqual(matchedRanges, [
     {
       entryKey: 'grave hag',
```

Comment:
Apparently, the card was renamed. It would avoid confusion to rename the entry key as well.